pkg/nuclide/file-tree/spec/FileTreeController-spec.js

```diff
@@ -66,6 +66,21 @@ describe('FileTreeController', () => {
     fileTreeController.destroy();
   });
 
+  it('checks if core:backspace and core:delete is registered on .nuclide-file-tree', () => {
+    var timesCalled = 0;
+    // Find div element
+    var el = fileTreeController._panelController._hostEl.getElementsByClassName('nuclide-file-tree')[0];
```

Comment:
To keep to the public API of the FileTreeController, go through `getTreeComponent` and use React's [TestUtils](https://facebook.github.io/react/docs/test-utils.html#findrendereddomcomponentwithclass):

```javascript
var {TestUtils} = require('react-for-atom').addons;

...

var el = TestUtils. findRenderedDOMComponentWithClass(
  fileTreeController.getTreeComponent(),
  'nuclide-file-tree'
);
```